spec/system/custom_sidebar_sections_spec.rb

```diff
@@ -34,6 +34,25 @@
     expect(page).to have_link("Sidebar Tags")
   end
 
+  it "allows the user to create custom section with external link" do
+    visit("/latest")
+    sidebar.open_new_custom_section
+
+    expect(section_modal).to be_visible
+    expect(section_modal).to have_disabled_save
```

Comment:
```suggestion
    expect(section_modal).to have_disabled_save_button
```

Same suggestion but I think this is clearer.